### DISCUSSION TITLE
Add Distance Snap for Catch Editor Sliders/Juice Streams

### DISCUSSION THREAD

**Discussion by rcfox2** [ppy/osu#18471] [Ideas] [answered]
Since a recent update, slider speed has been completely unlocked in the catch editor. See video:

https://user-images.githubusercontent.com/68173946/170848247-a42450f3-4342-4b4e-95d3-a794335c4821.mp4

While I am in favor of this change, I used the slider speed feature to keep my sliders at around sprinting speed. Since you can use distance snap to snap normal notes to sprinting speed, it would be nice if you could see distance snap when placing and selecting a slider, but the only way to distance snap sliders at the moment is to place a note in the same spot as the start of the slider, and compare the distance snap from it:
![image](https://user-images.githubusercontent.com/68173946/170848299-47fc1616-14ec-461b-97cc-f369a0505d28.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/18491
